src/main.rs

```diff
@@ -20,6 +22,22 @@ fn main() -> Result<()> {
     }
     env_logger::init();
 
+    StdErr::deprecation_warning(&format!(
+        r#"{} {} {}
+
+You are using an outdated version of Wrangler (v{}).
+Please upgrade to v2 ({}).
+Detailed migration instructions can be found here:
+{}
+"#,
+        emoji::WARN,
+        styles::warning("DEPRECATION WARNING"),
```

Comment:
Rather than "Deprecation" 
```suggestion
        styles::warning("UPGRADE AVAILABLE"),
```